joomla/joomla-cms
# [com_contact] review config global default options

### Summary of Changes

This PR intends to normalize onfig global default options in com_contact.

The method used was:
1. Review cs of the com_contact config.xml file - also add missing default values
2. Check the current params in installation joomla.sql mysql
3. Add missing defaults to install sql

### Testing Instructions

0. Code review
1. Apply patch
2. Delete configuration.php
3. Install Joomla (no sample)
4. Create a contact and check all "Use Global" have the global value.

### Documentation Changes Required

None.

### Added/changed defaults in the xml

changed:
- history_limit: 10 (was 5 - uniformize with articles)

added:
- presentation_style: sliders (was empty - but sliders was the first choice)
- media: *empty*
- show_user_custom_fields: -1 (was empty - but was the first choice)
- linka_name, linkb_name, linkc_name, linkd_name, linke_name: *empty*
- icon_address, icon_email, icon_telephone, icon_mobile, icon_fax, icon_misc: *empty*
- banned_email, banned_subject, banned_text: *empty*
- redirect: *empty*
